app/controllers/home_controller.rb

```diff
@@ -41,7 +41,6 @@ def search
     @tools = Tool.search(@query)
     @organizations = Organization.search(@query)
     @organization_aliases = OrganizationAlias.search(@query)
-    @documents = Document.search(@query)
```

Comment:
I didn't even know this functionality existed. I'll make a note to let people know to search for documents in our shared Google Drive now.